.github/workflows/deploy_artifacts.yaml

```diff
@@ -111,10 +117,11 @@ jobs:
           DEPLOY_TOKEN: ${{ github.ref_type == 'tag' && secrets.COLLAB_DEPLOY_TOKEN  || secrets.SNAPSHOT_DEPLOY_TOKEN }}
 
   deploy_image:
-    needs: check_tag
+    needs: set_changelist
     if: ${{ inputs.createDockerImage && inputs.quayRepository != '' }}
-    uses: dockstore/workflow-actions/.github/workflows/deploy_image.yaml@main
+    uses: dockstore/workflow-actions/.github/workflows/deploy_image.yaml@seab-6860/image-changelist
```

Comment:
Made this same comment in Kathy's PR, but I think we should start tagging in `dockstore/workflow-actions` and reference actions accordingly (can wait for this PR).